Marlin/src/pins/stm32f4/pins_FYSETC_CHEETAH_V30.h

```diff
@@ -41,12 +41,12 @@
 //
 // Z Probe
 //
-//#if ENABLED(BLTOUCH)
-//  #error "You need to set jumper to 5V for BLTouch, then comment out this line to proceed."
-//  #define SERVO0_PIN                      PA0
-//#elif !defined(Z_MIN_PROBE_PIN)
-//  #define Z_MIN_PROBE_PIN                 PA0
-//#endif
+#if ENABLED(BLTOUCH)
+  #error "You need to set jumper to 5V for BLTouch, then comment out this line to proceed."
+  #define SERVO0_PIN                        PA0   // PROBE
+#elif !defined(Z_MIN_PROBE_PIN)
+  #define Z_MIN_PROBE_PIN                   PA0
+#endif
```

Comment:
Do you know why this was commented out before? I assume something it it was incorrect or problematic?

It seems unrelated to the rest of the changes.

Reply:
This isn't the first time we've seen quirkiness in FYSETC pins and instead of removing the disabled defines, I pulled up their board info and verified / enabled these defines and added PCB silkscreen labels as comments like we usually do).